src/shell.ts

```diff
@@ -536,4 +536,8 @@ export class Shell {
     async reset() {
         this.pbSession = null
     }
+    async offerSub() {
```

Comment:
This function is not doing enough to merit a function. Please copy the notification to the only place we call this function and remove the function.

Reply:
I'm refactoring it myself